extensions/ql-vscode/src/config.ts

```diff
@@ -647,3 +647,16 @@ export class MockGitHubApiConfigListener
 export function getMockGitHubApiServerScenariosPath(): string | undefined {
   return MOCK_GH_API_SERVER_SCENARIOS_PATH.getValue<string>();
 }
+
+/**
+ * Enables features that are specific to the codespaces-codeql template workspace from
+ * https://github.com/github/codespaces-codeql.
+ */
+export const CODESPACES_CODE_TOUR = new Setting(
```

Comment:
Minor: do you want a name that applies to the template in general, regardless of the code tour?